manifests/m/MSYS2/MSYS2/20221028/MSYS2.MSYS2.locale.en-US.yaml

```diff
@@ -0,0 +1,33 @@
+# Created with YamlCreate.ps1 v2.2.0 using InputObject 🤖 $debug=QUSU.CRLF..7-2-7
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: msys2.msys2
+PackageVersion: "20221028"
+PackageLocale: en-US
+Publisher: The MSYS2 Developers
+PublisherUrl: https://www.msys2.org/
+PublisherSupportUrl: https://www.msys2.org/contact/
+# PrivacyUrl:
+Author: The MSYS2 Developers
+PackageName: MSYS2 Installer
+PackageUrl: https://github.com/msys2/msys2-installer
+License: BSD-3-Clause
+LicenseUrl: https://github.com/msys2/msys2-installer/blob/main/LICENSE
+Copyright: Copyright 2022 MSYS2 devs
+# CopyrightUrl:
+ShortDescription: Software Distribution and Building Platform for Windows
+Description: MSYS2 is a collection of tools and libraries providing you with an easy-to-use environment for building, installing and running native Windows software.
+# Moniker:
+# Tags:
+# Agreements:
+# ReleaseNotes:
```

Comment:
```suggestion
ReleaseNotes: |-
  - At the end run a ucrt64 shell instead of a mingw64 one, see https://github.com/msys2/MINGW-packages/issues/13371
  - The usual round of package updates
```